baremetal_network_provisioning/ml2/hp_network_provisioning_driver.py

```diff
@@ -203,8 +210,8 @@ def _frame_switch_url(self, switch_id):
         url = self.base_url + '/' + switch_id
         return url
 
-    def _get_port_pay_load(self, port_dict):
-        """form  port payload for SDN controller REST request."""
+    def _get_port_pay_load(self, port_dict, include_seg_id=None):
+        """Form  port payload for SDN controller REST request."""
```

Comment:
please retrive segmentation_id and access_type directly from port_dict itself without intermediate bind_port_dict.